nfl_data_py/tests/nfl_test.py

```diff
@@ -8,12 +8,22 @@ def test_is_df_with_data(self):
         s = nfl.import_pbp_data([2020])
         self.assertEqual(True, isinstance(s, pd.DataFrame))
         self.assertTrue(len(s) > 0)
+
+    def test_is_df_with_data_thread_requests(self):
+        s = nfl.import_pbp_data([2020, 2021], thread_requests=True)
+        self.assertEqual(True, isinstance(s, pd.DataFrame))
+        self.assertTrue(len(s) > 0)
 		
 class test_weekly(TestCase):
     def test_is_df_with_data(self):
         s = nfl.import_weekly_data([2020])
         self.assertEqual(True, isinstance(s, pd.DataFrame))
         self.assertTrue(len(s) > 0)
+
+    def test_is_df_with_data_thread_requests(self):
+        s = nfl.import_weekly_data([2020, 2021], thread_requests=True)
+        self.assertEqual(True, isinstance(s, pd.DataFrame))
+        self.assertTrue(len(s) > 0)
         
```

Comment:
If you're feeling ambitious, you could assert that this method completes at least as fast as the non-threaded case 😄